apps/taiga-lumbermill/src/dashboards/login/login.component.ts

```diff
@@ -0,0 +1,63 @@
+import {CommonModule} from '@angular/common';
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {
+    FormControl,
+    FormGroup,
+    FormsModule,
+    ReactiveFormsModule,
+    Validators,
+} from '@angular/forms';
+import {
+    TuiAppearance,
+    TuiButton,
+    TuiError,
+    TuiLabel,
+    TuiLink,
+    TuiTitle,
+} from '@taiga-ui/core';
+import {TUI_VALIDATION_ERRORS, TuiCheckbox, TuiFieldErrorPipe} from '@taiga-ui/kit';
+import {TuiCardLarge, TuiHeader} from '@taiga-ui/layout';
+import {TuiInputModule, TuiInputPasswordModule} from '@taiga-ui/legacy';
+
+@Component({
+    standalone: true,
+    selector: 'lmb-login',
+    imports: [
+        CommonModule,
+        FormsModule,
+        ReactiveFormsModule,
+        TuiAppearance,
+        TuiButton,
+        TuiCardLarge,
+        TuiCheckbox,
+        TuiError,
+        TuiFieldErrorPipe,
+        TuiHeader,
+        TuiInputModule,
+        TuiInputPasswordModule,
+        TuiLabel,
+        TuiLink,
+        TuiTitle,
+    ],
+    templateUrl: './login.component.html',
+    styleUrl: './login.component.less',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [
+        {
+            provide: TUI_VALIDATION_ERRORS,
+            useValue: {
+                required: "Value can't be empty",
+                email: 'Value should be email',
+            },
+        },
+    ],
+})
+export class LoginComponent {
+    protected readonly form = new FormGroup({
+        email: new FormControl('', [Validators.required, Validators.email]),
+        password: new FormControl('', [Validators.required]),
+        rememberMe: new FormControl(false),
+    });
+
+    protected onSubmit(): void {}
```

Comment:
let's add some interactivity on submit
loader button and redirect to main page

Reply:
design